release_docs/RELEASE.txt

```diff
@@ -116,20 +115,20 @@ New Features
       arch dependent, and not easily readable by a human being. They are
       nevertheless searched for in the includes directories by gfortran
       (in directories specified with -I).
-
+// Should arch be architecture in the above paragraph? // 
```

Comment:
yes

Reply:
I fixed this by editing and re-committing.